jkarenzi/task-master-fe
# Feat: Implement user authentication and signup

### What does this PR do?
- This PR implements user authentication and sign up.

### Description of the task to be completed
- [x] Implement signup and login pages
- [x] Implement 2FA page and email verification page
- [x] Implement email verification and 2FA email template designs
- [x] Integrate the above pages with backend

### How can this be manually tested?
- Create a new user by signing up
- Verify your email
- Login with created user

### Test screenshot
<img width="716" alt="fee" src="https://github.com/jkarenzi/task-master-fe/assets/109604986/107b113c-463d-4139-993b-bacd5975fbaa">
<img width="725" alt="fee2" src="https://github.com/jkarenzi/task-master-fe/assets/109604986/6b853a0f-021e-4161-89db-8255deaef562">

### What are the relevant pivotal trackers/story id?
- #3
